version.go

```diff
@@ -0,0 +1,5 @@
+package aiven
+
+func version() string {
+	return "v1.1.0"
```

Comment:
Instead of having this file checked into git it could be automatically generated by some `make` target based on `git describe --tags` output. As it is now I'm pretty sure people forget to update this all the time. (Having the commit id included in the version when current commit does not exactly match any tag is OK.)

Reply:
Come to think of that, I'm not sure how it works when this library is imported into the Terraform provider project. If that just gets files that are checked into Github without invoking any `make` targets then that's not good. https://www.atatus.com/blog/golang-auto-build-versioning/ also mentions one approach for being able to set a version, which could perhaps even be used to override the generic aiven-go-client version with Terraform provider version from that project, which would be possibly more useful than the plain go client version.

Reply:
I agree that hard coded version is not optimal but it seems like it is not possible to pass the build flag as a configuration that will be applied from `go get`, for eg. The build flag only works if we distribute binaries.